ext/phar/phar_object.c

```diff
@@ -512,6 +523,14 @@ PHP_METHOD(Phar, mount)
 	}
 
 	zend_throw_exception_ex(phar_ce_PharException, 0, "Mounting of %s to %s failed", path, actual);
+
+finish: ;
```

Comment:
`;` ? :)

Reply:
Having a goto label with nothing after it is illegal, which would happen on linux without this hack.

Reply:
Indeed.